README.md

```diff
@@ -123,13 +111,17 @@ Moreover, descriptions of the configuration (resource) parameters are included i
 
 The steps detailed below are essentially those performed by `parallel_build.csh` in Step 3 above. Either method should yield identical builds.
 
-##### Checkout externals
+#### mepo
+
+The GEOSldas is comprised of a set of sub-repositories. These are
+managed by a tool called [mepo](https://github.com/GEOS-ESM/mepo). To
+clone all the sub-repos, you can run `mepo clone` inside the fixture:
 ```
 cd GEOSldas
-checkout_externals
+mepo clone
```

Comment:
Do we need to add "mepo init" here?  This part of the instructions is for the case where users want to bypass parallel_build.csh (for whatever reason).  In parallel_build.csh, the corresponding commands are:
```
mepo init
mepo clone
```
I might be missing something, so sorry if the question is silly.

Reply:
@gmao-rreichle Technically no, but I can add it. If you run `mepo clone` without `mepo init`, `mepo` will run the init step for you. It was needed to have a `mepo clone -b tag URL` all-in-one command. Because of that, the commands that run all sort of learned how to call what was needed.

But it can't hurt to have it, so I've added it.